Utilities/bin_patch.h

```diff
@@ -20,7 +20,7 @@ namespace patch_key
 	static const std::string version = "Version";
 }
 
-static const std::string patch_engine_version = "1.2";
+inline static const std::string patch_engine_version = "1.2";
```

Comment:
will constexpr std::string_view work?

Reply:
id imagine so :)

Reply:
@elad335  should i change it to contexpr?

Reply:
constexpr implies inline.

Reply:
I mean yes.